config/application.rb

```diff
@@ -22,5 +22,11 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
```

Comment:
@ski-climb that's straight from the rack-cors docs, but if you want to change it that's fine. Do you just want me to specify the port numbers?